RecoTracker/MkFitCore/interface/Config.h

```diff
@@ -0,0 +1,433 @@
+#ifndef RecoTracker_MkFitCore_interface_Config_h
+#define RecoTracker_MkFitCore_interface_Config_h
+
+#include <algorithm>
+#include <cmath>
+#include <string>  // won't compile on clang gcc for mac OS w/o this!
```

Comment:
> is a mac build of CMSSW supported?

Mac is not supported, nor is support planned. Even much simpler FWLite has become more and more difficult to support for Mac.

Reply:
Well ... but it still makes sense to support mkfit standalone build on osx.
Still, I agree to removing this (and others like it) and then revive the osx build at a later time / as needed.

Reply:
Addressed in Round 2 follow-up PR trackreco/cmssw#76 commit https://github.com/trackreco/cmssw/pull/76/commits/6f728696277e15a8502cb9e080061ad7d504f604

I dropped the comment; the include is formally needed because std::string is used later in the file. So, the comment about OS is somewhat out of place